README.md

```diff
@@ -27,42 +27,42 @@ This repository will be home to the source code for the `mediapipe_vision`, `med
     </tr>
     <tr>
         <td>Classification</td>
-        <td align="center"><img height="16" width="16" src="https://raw.githubusercontent.com/google/flutter-mediapipe/main/assets/yes.png" /></td>
-        <td align="center"><img height="16" width="16" src="https://raw.githubusercontent.com/google/flutter-mediapipe/main/assets/yes.png" /></td>
-        <td align="center"><img height="16" width="16" src="https://raw.githubusercontent.com/google/flutter-mediapipe/main/assets/no.png"/></td>
-        <td align="center"><img height="16" width="16" src="https://raw.githubusercontent.com/google/flutter-mediapipe/main/assets/no.png"/></td>
-        <td align="center"><img height="16" width="16" src="https://raw.githubusercontent.com/google/flutter-mediapipe/main/assets/yes.png" /></td>
-        <td align="center"><img height="16" width="16" src="https://raw.githubusercontent.com/google/flutter-mediapipe/main/assets/no.png"/></td>
+        <td align="center">✅</td>
+        <td align="center">✅</td>
+        <td align="center">-</td>
```

Comment:
Does a red emoji make sense here instead of a single dash?

Reply:
I tend to think of an unfilled square as more 'missing' rather than an error state, but this is your readme :) I can try that out something like ❌ or ✖.

Reply:
Hmm, that's a good point. We can go with `-`.